lib/base-compiler.js

```diff
@@ -862,10 +859,9 @@ Please select another pass or change filters.`;
                 if (postProcess.length) {
                     return this.execPostProcess(result, postProcess, outputFilename, maxSize);
                 } else {
-                    return this.readFile(outputFilename).then(contents => {
-                        result.asm = contents.toString();
-                        return Promise.resolve(result);
-                    });
+                    const contents = await fs.readFile(outputFilename);
+                    result.asm = contents.toString();
+                    return Promise.resolve(result);
```

Comment:
As with the earlier comment, could this be a `return result` now? Or am I missing something (Still on the process of fully grasping async/await)

Reply:
It can be, yes.

Reply:
Thanks! :)